python/analyzers/treeVariables.py

```diff
@@ -20,7 +20,8 @@ def default():
 # event variables
 event_vars = [
     Variable('run', type=int),
-    Variable('event', lambda ev : ev.eventId, type=int),
+    Variable('lumi', lambda ev : ev.lumi, type=int),
+    Variable('event', lambda ev : ev.eventId, type=int), # in the new python version, 'long' and 'int' are both merged into 'int', so it should be taken care of here.
```

Comment:
new python version?
We're bound to use CMSSW's python for ntuple production, so there's no 'new python'